force-app/main/default/lwc/mVPNomineeBasicInfo/mVPNomineeBasicInfo.js

```diff
@@ -0,0 +1,27 @@
+import { LightningElement, api } from 'lwc';
+import getFormFields from '@salesforce/apex/GetContact.getFormFields';
+export default class MVPNomineeBasicInfo extends LightningElement {
+
+    // Flexipage provides recordId and objectApiName
+    @api recordId;
```

Comment:
We can not pick `recordId` from @api as we are planning to use this component on designation page,
use something else to fit in our requirement